lms/djangoapps/teams/static/teams/templates/team-profile.underscore

```diff
@@ -43,7 +43,7 @@
                 <span class="sr"><%- gettext("The language that team members primarily use to communicate with each other.") %></span>
                 <i class="icon fa fa-comment-o fa-fw" aria-hidden="true"></i>
                 <span>
-                    <%- gettext(language) %>
+                    <%= HtmlUtils.ensureHtml(gettext(language)) %>
```

Comment:
This change doesn't seem necessary. ``<%- gettext(language) %>`` will correctly escape the text.